### PR TITLE
[Android] handle CALL_FROM with null value

### PR DESCRIPTION
**Pull request by shiburagi**
Hi,
On this commit, I added **null checker** before replace "client:" on **handleCallIntent** method in **BackgroundCallJavaActivity** class.

### CODE REVIEW

**Comment by diegogarciar** [comment]
Looks good! could you explain in which scenario `call_from` can be null?

**Comment by cybex-dev** [comment]
"From" could be null if the 'client' e.g. external application processes calls (with their `makeCall`) function differently with alternative parameters names, which coincidentally arrives at TwilioVoice.

TwiML app could have:

See [voice-quickstart-server-node](https://github.com/twilio/voice-quickstart-server-node/blob/master/src/server.js#L71)'s `makeCall` function, which has no `From` parameter but could be written as this:
```
exports.handler = async (context, event, callback) => {
    // The recipient of the call, a phone number or a client
    var to = event.otherId; // usually 'To'
    var from = event.myId;// usually 'From'

    const dial = voiceResponse.dial({callerId: from || defaultCallerId});
    dial.client(to);
    ...
```


_A possible future addition provide a transform function to interpret these parameters instead of relying on hardcoded 'From' and 'To' parameters_

File: android/src/main/java/com/twilio/twilio_voice/BackgroundCallJavaActivity.java [hunk 1]
```diff
@@ -220,4 +221,4 @@ protected void onDestroy() {
         deactivateSensor();
     }
 
-}
\ No newline at end of file
+}
```
Comment (cybex-dev) [inline android/src/main/java/com/twilio/twilio_voice/BackgroundCallJavaActivity.java:224]: @shiburagi please remove these 2 lines, will merge changes into master for next twilio_voice update

Comment (cybex-dev) [reply android/src/main/java/com/twilio/twilio_voice/BackgroundCallJavaActivity.java:224]: @shiburagi

**Comment by cybex-dev** [comment]
This addition is no longer relevant to Android.

**Pull request closed without merging**